monitoring/bin/deploy_monitoring_viya.sh

```diff
@@ -7,6 +7,17 @@ cd "$(dirname $BASH_SOURCE)/../.."
 source monitoring/bin/common.sh
 source bin/openshift-include.sh
 
+# If openshift deployment, ensure user-workload monitoring is enabled
+if [ -n "$(kubectl get ns openshift-monitoring -o name 2>/dev/null)" ]; then
```

Comment:
chose `openshift-monitoring` namespace as that should only exist in the event of an openshift deployment with monitoring deployed. 

Any concerns about choosing this one?

Reply:
Is it even possible to deploy OpenShift without the built-in Prometheus?  Our tooling assumes that the built-in Prometheus is "there".  If it is possible that is not the case, we probably should be handling that scenario (but not in this current PR/effort).